modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -4,147 +4,58 @@
-|
-|
-
-|`storage:`
-|Configuration options that define the storage. All storage-related options must be placed under `storage` and not under the `allInOne` or other component options.
-|
-|
-
-|`template.distributor:`
-|Configuration options for the Tempo `distributor`.
-|
-|
-
-|`template.ingester:`
-|Configuration options for the Tempo `ingester`.
-|
-|
-
-|`template.compactor:`
-|Configuration options for the Tempo `compactor`.
-|
-|
-
-|`template.querier:`
-|Configuration options for the Tempo `querier`.
-|
-|
-
-|`template.queryFrontend:`
-|Configuration options for the Tempo `query-frontend`.
-|
-|
-
-|`template.gateway:`
-|Configuration options for the Tempo `gateway`.
-|
-|
-
-|===
-
-
-
-.Minimum required configuration
-
-The following is the required minimum for creating a {TempoShortName} deployment with the default settings:
-
-[source,yaml]
-----
-apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoStack
-metadata:
-  name: simplest
-spec:
-  storage: # <1>
-    secret:
-      name: minio
-      type: s3
-  resources:
-    total:
-      limits:
-        memory: 2Gi
-        cpu: 2000m
-  template:
-    queryFrontend:
-      jaegerQuery:
-        enabled: true
-        ingress:
-          type: route
-----
-<1> This section specifies the deployed object storage back end, which requires a created secret with credentials for access to the object storage.
+<1> API version to use when creating the object.
+<2> Defines the kind of Kubernetes object to create.
+<3> Data that uniquely identifies the object, including a `name` string, `UID`, and optional `namespace`. {product-title} automatically generates the `UID` and completes the `namespace` with the name of the project where the object is created.
+<4> Name of the TempoStack instance.
+<5> Contains all of the configuration parameters of the TempoStack instance. When a common definition for all Tempo components is required, it is defined under the `spec` node. When the definition relates to an individual component, it is placed under the `spec/template/<component>` node.
+<6> Storage is specified for instance deployment. See the installation page for information about storage options for the instance.
+<7> Defines the compute resources for the Tempo container.
+<8> Configuration for the replication factor.
+<9> Configuration options for retention of traces.
+<10> Configuration options for the Tempo `distributor`.
+<11> Configuration options for the Tempo `ingester`.
+<12> Configuration options for the Tempo `compactor`.
+<13> Configuration options for the Tempo `querier`.
+<14> Configuration options for the Tempo `query-frontend`.
+<15> Configuration options for the Tempo `gateway`.
+<16> Limits ingestion and query rates.
+<17> Defines ingestion rate limits.
+<18> Defines query rate limits.
+<19> Configures operands to handle telemetry data.
+<20> Configures search capabilities.
+<21> Defines whether or not the CR is managed by the Operator. The default value is `managed`.
```

Comment:
```suggestion
<21> Defines whether or not the `TempoStack` CR is managed by the Operator. The default value is `managed`.
```
Recommend adding CR name for clarity, but not required

Reply:
Thank you. :sweat_smile: This is a bit of a double-edged sword: if I read it, I'd think that the `TempoStack` CR is some different CR.